src/main/java/com/exception/IncorrectPlaceNumberException.java

```diff
@@ -0,0 +1,11 @@
+package com.exception;
+
+public class IncorrectPlaceNumberException extends Exception {
+    public IncorrectPlaceNumberException(String errorMessage) {
+        super(errorMessage);
+    }
+
+    public static void main(String[] args) {
+
+    }
+}
```

Comment:
тут цього не повинно бути
```suggestion
}
```